Misc/NEWS.d/next/Library/2024-07-30-14-46-16.gh-issue-87320.-Yk1wb.rst

```diff
@@ -0,0 +1,2 @@
+Don't crash :class:`code.InteractiveInterpreter` when calling
```

Comment:
"Crash" has a special meaning in Python. It means segmentation fault or other hard errors. This is not a crash.

Reply:
Ok. other suggestions? "Don't leave ..."? "Don't quit"?

Reply:
I do not know. Maybe "Handle exceptions ..."?

Reply:
I tried to make it clearer by adding a second sentence.